sklearn/ensemble/voting_classifier.py

```diff
@@ -61,6 +62,12 @@ class VotingClassifier(_BaseComposition, ClassifierMixin, TransformerMixin):
         The number of jobs to run in parallel for ``fit``.
         If -1, then the number of jobs is set to the number of cores.
 
+    flatten_transform : bool, optional (default='default')
+        Affects shape of transform output only when voting='soft'
+        If voting='soft' and flatten_transform=True, transform method returns
+        matrix with shape (n_samples, n_classifiers * n_classes) instead of
```

Comment:
Maybe say what the current default behavior is and that it will change in the future. You can also use a ``versionadded`` sphinx directive here.